src/OmniSharp.Script/CachingScriptMetadataResolver.cs

```diff
@@ -47,7 +51,7 @@ public override ImmutableArray<PortableExecutableReference> ResolveReference(str
                 return DirectReferenceCache[key];
             }
 
-            var result = _defaultRuntimeResolver.ResolveReference(reference, baseFilePath, properties);
+            var result = _defaultReferenceResolver.ResolveReference(reference, baseFilePath, properties);
```

Comment:
it's not visible what happens here for nuget packages, can you explain a little? do they get installed? what if there is no network? what if there is no `nuget` in the user's path? what's the lock file?